package.json

```diff
@@ -114,6 +113,8 @@
     "snapshotSerializers": [
       "jest-serializer-html"
     ],
+    "testEnvironment": "jsdom",
+    "testRunner": "jest-jasmine2",
```

Comment:
jest 27 changed default behaviour, this gets us the legacy behaviours our tests depend on.